apps/system/test/unit/accessibility_test.js

```diff
@@ -25,6 +26,28 @@ suite('system/Accessibility', function() {
     options: { pattern: [40], isKey: false }
   };
 
+  var vcChangeHintsEnabledDetails = {
+      eventType: 'vc-change',
+      data: [{string: 'link'}],
+      options: {
+        hints: [{string: 'link-hint'}]
+      }
+  };
+
+  var aDataPlusHints = [
+    {string: 'link'},
+    {string: 'link-hint'}
+  ];
+
+  // Can't get Chai or Sinon to match this function when compared
+  // to the argument list.  Will remove if unusable
+  /*var hintsCallback = function hintsCallback() {
+    if (options.hints) {
+      this.setHintsTimeout(aDetails.data.concat(options.hints));
+    }
+  }.bind(accessibility);*/
```

Comment:
As you can see from this comment, I'm having trouble getting Chai or Sinon to equate this defined function (or even an inline replica) to match the inline function given as an argument in accessibility.js (see more below)